SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -385,6 +389,34 @@ TArray<FWorkerComponentData> EntityFactory::CreateEntityComponents(USpatialActor
 	return ComponentDatas;
 }
 
+TArray<FWorkerComponentData> EntityFactory::CreateSkeletonEntityComponents(AActor* Actor)
+{
+	TArray<FWorkerComponentData> ComponentDatas = CreateMinimalEntityComponents(Actor);
+
+	// LB components also contain authority delegation, giving this worker ServerAuth.
+	WriteLBComponents(ComponentDatas, Actor);
+
+	// Empty RPC components.
+	Algo::Transform(SpatialRPCService::GetRPCComponents(), ComponentDatas, &ComponentFactory::CreateEmptyComponentData);
```

Comment:
This seems a bit odd. Why not just add N empty component data objects?

Reply:
Oh I see you're getting the IDs. We could also make it that the service just creates the components and does that as we might want to populate them too. But this is fine.